pkg/maps/ctmap/ctmap_privileged_test.go

```diff
@@ -836,3 +860,115 @@ func (k *CTMapPrivilegedTestSuite) TestOrphanNatGC(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(len(buf), Equals, 0)
 }
+
+// TestCount checks whether the CT map batch lookup dumps the count of the
+// entire map reliably, while the map is changing in the background.
+func (k *CTMapPrivilegedTestSuite) TestCount(c *C) {
```

Comment:
No need to extend gocheck suites:
```suggestion
func TestCount(t *testing.T) {
    testutils.PrivilegedTest(t)
```